app/src/main/res/layout-sw600dp-port/hints_summary.xml

```diff
@@ -30,6 +30,7 @@
 
       <TextView
         android:id="@+id/hint_title"
+        style="@style/TextViewStart"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
```

Comment:
We are using `TextViewStart` and the `layout_gravity=center`. Should we use `TextViewCenter`

Reply:
As per the mock https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/e5d3beb6-6f0a-47cf-bd40-4ef00ef5e53d/specs/ text is aligned at the start. `layout_gravity` becomes unused here. Removed it.